recipes/setuptools/bld.bat

```diff
@@ -0,0 +1,8 @@
+python setup.py install
```

Comment:
Nope. Seems like this file doesn't help. Weird that it has the same as the script we are recommended to download.